contentcuration/contentcuration/settings.py

```diff
@@ -68,7 +70,6 @@
 
 INSTALLED_APPS = (
     'contentcuration.apps.ContentConfig',
-    'django.contrib.admin',
```

Comment:
It's fine to leave this in all the time, it's only the URLs we need to exclude.

Reply:
Done